index.html

```diff
@@ -1,83 +1,94 @@
 <!doctype html>
 <html ng-app="app1">
-	<head>
-		<link rel="stylesheet" href="styles/namelist.css">
-		<link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
-		<link rel="stylesheet" href="styles/general.css">
-		<title>Student reports generator</title>
-	</head>
-	<body>
-		<div class="container">
-			<div class="page-header">
-				<h1>Student reports generator</h1>
-			</div>
+<head>
```

Comment:
Depends what convention you follow. My IDE autoformatted it this way and I didn't event think about it. I'll change it back.